components/atoms/BenefitCardHeaderActive.js

```diff
@@ -7,11 +7,11 @@ export default function BenefitCardHeaderActive(props) {
   const t = props.locale === 'en' ? en : fr
 
   function viewPaymentHistory() {
-    console.log('View payment history button is Clicked')
+    window.open('./dashboard', 'ViewMyPayments')
```

Comment:
How come this was changed from `console.log()`?

Reply:
I removed <a></a> and used window.open() to link to dashboard page.
console.log() had been added to show button works. We don't need it anymore. Click on button refresh the page.

Reply:
Can't you use the Next's router instead of `window.open()`?

Reply:
I removed window,open and used Next's router